Makefile.features_modules

```diff
@@ -0,0 +1,27 @@
+# Add modules implementing used features
```

Comment:
Do you mind splitting out this cleanup in a different PR?

Reply:
Or I could do it, but to cleanup the erroneous modules I added in #13639

Reply:
I can do so. But this PR would depend on this, as this is needed to work.

But let me first get this working ;-)

Reply:
Yep, its just that since you are fixing an unrelated issue, it will make testing easier if this is already in, the output of `makefiles/dependencies_debug.inc.mk` and other such tools will be cleaner :)